easybuild/easyconfigs/o/OpenSceneGraph/OpenSceneGraph-3.6.5-foss-2022a.eb

```diff
@@ -0,0 +1,51 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author:    Jiri Furst <jiri.furst@gmail.com>
+###
+
+easyblock = 'CMakeMake'
+
+name = 'OpenSceneGraph'
+version = '3.6.5'
+
+homepage = 'http://www.openscenegraph.org/'
+description = """The OpenSceneGraph is an open source high performance 3D graphics toolkit, 
+used by application developers in fields such as visual simulation, games, virtual reality, 
+scientific visualization and modelling. Written entirely in Standard C++ and OpenGL it runs 
+on all Windows platforms, OSX, GNU/Linux, IRIX, Solaris, HP-Ux, AIX and FreeBSD operating 
+systems. The OpenSceneGraph is now well established as the world leading scene graph 
+technology, used widely in the vis-sim, space, scientific, oil-gas, games and virtual 
+reality industries."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['OpenSceneGraph-%(version)s.tar.gz']
+checksums = ['aea196550f02974d6d09291c5d83b51ca6a03b3767e234a8c0e21322927d1e12']
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('pkg-config', '0.29.2'),
```

Comment:
```suggestion
    ('pkgconf', '1.8.0'),
```

Reply:
`pkg-config` is no longer maintained, so for `2022a` onwards we prefer to use `pkgconf` where possible